### DISCUSSION TITLE
How to extract MSI

### DISCUSSION THREAD

**Discussion by anderlli0053** [ScoopInstaller/Scoop#5101] [Help and Questions] [answered]
How to properly extract `.msi `archive from the .zip and then extract it?

There is nothing like `Expand-MsiArchive` option as opposed to regular `"Expand-Archive"` and `"Expand-7zipArchive"`.

I know that there is lessmsi tool (Les Msierable), but if i can call that tool, how can i call it properly to extract?

**Comment by r15ch13** [comment, marked as answer]
Sure there is `Expand-MsiArchive`
https://github.com/ScoopInstaller/Scoop/blob/4a31bd330244f7f89f16208cdddda3f9edac2d65/lib/decompress.ps1#L127

Example:
https://github.com/ScoopInstaller/Main/blob/c70fd214b19296e6e67045f0da142b0d9304f972/bucket/sliksvn.json#L6-L19

**Comment by anderlli0053** [comment]
@r15ch13  Thank you, i do not know how i didn't found that in the docs though..
